### PR TITLE
Check both python versions lints

### PR DESCRIPTION
**Pull request by o01eg**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,20 @@ jobs:
...
+      dist: bionic
+      language: python
+      python: 3.5
+      before_install:
+        - pip install flake8-putty==0.4.0
+      before_script:
+        - cd default/python
+      script:
+        - flake8
+        - python -m compileall -q .
```
Comment (adrianbroher) [inline .travis.yml]: Why are you running python2?

File: .travis.yml [hunk 2]
```diff
@@ -19,6 +19,20 @@ jobs:
         - python -m compileall -q .
       env:
         - NAME="Lint Python script"
+    - stage: lint
+      os: linux
+      dist: bionic
+      language: python
+      python: 3.5
+      before_install:
+        - pip install flake8-putty==0.4.0
```
Comment (adrianbroher) [inline .travis.yml]: Why are you installing into the python2 site-packages?

File: .travis.yml [hunk 3]
```diff
@@ -19,6 +19,20 @@ jobs:
...
+      before_script:
+        - cd default/python
+      script:
+        - flake8
+        - python -m compileall -q .
+      env:
+        - NAME="Lint Python script"
```
Comment (adrianbroher) [inline .travis.yml]: You are linting python3 syntax, that should be reflected in the name.

**Review by adrianbroher — CHANGES_REQUESTED** [review]
The change only runs Python 2 linting.

File: .travis.yml (hunk 3 above)
Comment (adrianbroher) [reply .travis.yml]: Also the env section was just a workaround for the at-that-time missing [job name](https://docs.travis-ci.com/user/build-matrix/#job-names) feature.

File: .travis.yml (hunk 1 above)
Comment (o01eg) [reply .travis.yml]: Fixed to use python3

File: .travis.yml (hunk 2 above)
Comment (o01eg) [reply .travis.yml]: Fixed to use python3

File: .travis.yml (hunk 3 above)
Comment (o01eg) [reply .travis.yml]: Fixed names to distinguish both.

**Comment by adrianbroher** [comment]
LGTM.

**Pull request merged**